packages/react-utilities/src/hooks/useTriggerElement.test.tsx

```diff
@@ -0,0 +1,108 @@
+import { fireEvent, render } from '@testing-library/react';
+import * as React from 'react';
+
+import { useTriggerElement } from './useTriggerElement';
+
+type TestTriggerProps = React.HTMLAttributes<unknown> & {
+  children: React.ReactElement;
+  onCustom?: () => void;
+};
+
+const TestTrigger = React.forwardRef<unknown, TestTriggerProps>((props, ref) => {
```

Comment:
nit: this test component really isn't necessary if you use `@testing-library/react-hooks` and just `render` the resulting hook call. It's what was done in the original `useTriggerElement` tests

Reply:
I would like to test components instead the hook itself:
- it's clearer how to use it
- it gives some space to refactor interfaces without refactoring tests